Firmware/LoRaSerial_Firmware/Commands.ino

```diff
@@ -12,7 +12,6 @@ void checkCommand()
   trimCommand(); //Remove any leading whitespace
 
   commandBuffer[commandLength] = '\0'; //Terminate buffer
-
```

Comment:
I don't know why the diff is showing a huge change in the commands.ino. Please rebase or clean as needed.